test/data/.sgnd/config/genesis.json

```diff
@@ -68,7 +68,8 @@
     "slash": {
       "params": {
         "signed_blocks_window": "20",
-        "min_signed_per_window": "0.500000000000000000",
+        "penalty_delegator_size": "1",
```

Comment:
`1` is usually a special case. Better to test with size 2. Then apply 3 delegators and test that the penalty can be split into two batches, first has 2 delegators, can second has 1 delegator